bin/gh-pages

```diff
@@ -10,12 +10,16 @@ program
       'base directory for all source files')
   .option('-s, --src <src>',
       'pattern used to select which files should be published', '**/*')
+  .option('-r, --repo <repo>',
+      'URL of the repository you\'ll be pushing to',
+      'URL of the `origin` remote of the current directory')
```

Comment:
You don't need a third arg here otherwise it will be used as a default value if --repo is not specified